parts/kubernetesmasteraddons-kubernetes-dashboard-deployment.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   labels:
     kubernetes.io/cluster-service: "true"
     k8s-app: kubernetes-dashboard
-    version: v1.5.1
```

Comment:
@amanohar `version` has been removed from this yaml. `image` is parameterized to the correct version and that is what matters.